mysql-8.0/docker-entrypoint.sh

```diff
@@ -59,20 +77,13 @@ _check_config() {
 # latter only show values present in config files, and not server defaults
 _get_config() {
 	local conf="$1"; shift
-	"$@" --verbose --help --log-bin-index="$(mktemp -u)" 2>/dev/null | awk '$1 == "'"$conf"'" { print $2; exit }'
+	"$@" --verbose --help --log-bin-index="$(mktemp -u)" 2>/dev/null \
+		| awk '$1 == "'"$conf"'" && /^[^ \t]/ { sub(/^[^ \t]+[ \t]+/, ""); print; exit }'
+	# match "datadir      /some/path with/spaces in/it here" but not "--xyz=abc\n     datadir (xyz)"
 }
 
 # allow the container to be started with `--user`
 if [ "$1" = 'mysqld' -a -z "$wantHelp" -a "$(id -u)" = '0' ]; then
-	# Docksal: copy custom settings (if mounted) from /var/www/.docksal/etc/mysql/my.cnf and fix permissions
```

Comment:
This custom block is used to inject Docksal db settings overrides. Please restore.

Reply:
with old docker-entrypoint.sh mysql server could not start

Reply:
custom code is restored